src/main/java/org/apache/rocketmq/dashboard/controller/DlqMessageController.java

```diff
@@ -70,4 +76,30 @@ public void exportDlqMessage(HttpServletResponse response, @RequestParam String
             throw new ServiceException(-1, String.format("export dlq message failed!"));
         }
     }
+
+    @PostMapping(value = "/batchResendDlqMessage.do")
+    @ResponseBody
+    public Object batchResendDlqMessage(@RequestBody List<DlqMessageRequest> dlqMessages) {
+        return dlqMessageService.batchResendDlqMessage(dlqMessages);
+    }
+
+    @PostMapping(value = "/batchExportDlqMessage.do")
+    public void batchExportDlqMessage(HttpServletResponse response, @RequestBody List<DlqMessageRequest> dlqMessages) {
+        List<DlqMessageExcelModel> dlqMessageExcelModelList = new ArrayList<>(dlqMessages.size());
+        for (DlqMessageRequest dlqMessage : dlqMessages) {
+            try {
+                String topic = MixAll.DLQ_GROUP_TOPIC_PREFIX + dlqMessage.getConsumerGroup();
+                MessageExt messageExt = mqAdminExt.viewMessage(topic, dlqMessage.getMsgId());
+                DlqMessageExcelModel excelModel = new DlqMessageExcelModel(messageExt);
+                dlqMessageExcelModelList.add(excelModel);
+            } catch (Exception e) {
+                log.error("Failed to query message by Id:{}", dlqMessage.getMsgId());
```

Comment:
Maybe returning the failed record with message-id and fail reason is better than just ignoring the exception.
```java
DlqMessageExcelModel excelModel = new DlqMessageExcelModel();
excelModel.setMessageId(messageId);
excelModel.setException(error);
dlqMessageExcelModelList.add(excelModel);
```